cairo/src/matrices.rs

```diff
@@ -7,14 +7,14 @@ use libc::c_double;
 #[repr(C)]
 #[derive(Debug, Clone, Copy, PartialEq)]
 pub struct Matrix {
```

Comment:
Should maybe be a `#[repr(transparent)]pub struct Matrix(ffi::cairo_matrix_t)`? Same for the others